roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -261,7 +261,7 @@ kiali_defaults:
       namespaces: []
       refresh_interval: "15s"
     validations:
-      ignore: []
+      ignore: ["KIA1201"]
```

Comment:
because we are changing the defaults, update the exampe kiali_cr.yaml which is the way we document for users what the defaults are. So you want to make a similar change here: https://github.com/kiali/kiali-operator/blob/master/deploy/kiali/kiali_cr.yaml#L911

Reply:
Right, thanks for the heads up.